spec/presenters/thread_spec.rb

```diff
@@ -80,7 +80,7 @@
       it "handles with_responses=false" do
         @threads_with_num_comments.each do |thread, num_comments|
           is_endorsed = num_comments > 0 && endorse_responses
-          hash = ThreadPresenter.new(thread, @reader, false, num_comments, is_endorsed).to_hash
+          hash = ThreadPresenter.new(thread, @reader, false, num_comments, is_endorsed).to_hash(false, 0, nil, false)
```

Comment:
For kicks, we should also test that `responses=false` and `recursive=true` doesn't do anything unexpected.  I think it should behave the same as `responses=false` and `recursive=false`, since it's not getting any results, so you could just loop the two checks here.